examples/GUI/GuiTools.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GuiTools
+ * Description: Classe pour créer un menu.
+ */
+
+/* global viewerDiv*/
+
+function GuiTools(api,domId) {
+    //Constructor
```

Comment:
Avoid useless comments. Same comment applies to the comments at the top of the file